staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -35,12 +35,12 @@ type Versioner interface {
 	// UpdateObject sets storage metadata into an API object. Returns an error if the object
 	// cannot be updated correctly. May return nil if the requested object does not need metadata
 	// from database.
-	UpdateObject(obj runtime.Object, resourceVersion uint64) error
+	UpdateObject(obj runtime.Object, etcdResourceVersion uint64) error
```

Comment:
As @liggitt mentioned on my PR, we shouldn't mention etcd specifically in this layer. Maybe "storageBackendResourceVersion" (ugh).

Reply:
sure that sounds good to me. I went back and forth on the names of these for a while and couldn't really find any that were super obvious.  but I think it should be the same throughout the code, so at a glance the variable name can tell you what value it should contain.